dom.bs

```diff
@@ -5194,10 +5217,9 @@ To <dfn export id=concept-element-attributes-set>set an attribute</dfn> given an
 
  <li><p>If <var>oldAttr</var> is <var>attr</var>, return <var>attr</var>.
 
- <li><p>If <var>oldAttr</var> is non-null, <a lt="remove an attribute">remove</a> it from
- <var>element</var>.
-
- <li><p><a lt="append an attribute">Append</a> <var>attr</var> to <var>element</var>.
+ <li><p>If <var>oldAttr</var> is non-null, <a lt="replace an attribute">replace</a> it
+  by <var>attr</var> in <var>element</var>. Otherwise <a lt="append an attribute">append</a>
+  <var>attr</var> to <var>element</var>.
```

Comment:
The otherwise can still be its own step. "Otherwise, append _attr_ to _element_."